docs/source/installation/jupyter.rst

```diff
@@ -0,0 +1,101 @@
+Jupyter Notebooks
+=================
+
+
+Binder
+------
+
+`Binder <https://mybinder.readthedocs.io/en/latest/>`__ is an online
+platform that hosts shareable and customizable computing environments
+in the form of Jupyter notebooks. Manim ships with a built-in ``%%manim``
+Jupyter magic command which makes it easy to use in these notebooks.
+
+To see an example for such an environment, visit our interactive
+tutorial over at https://try.manim.community/.
+
+It is relatively straightforward to prepare your own notebooks in
+a way that allows them to be shared interactively via Binder as well:
+
+#. First, prepare a directory containing one or multiple notebooks
+   which you would like to share in an interactive environment. You
+   can create these notebooks by using Jupyter notebooks with a
+   local installation of Manim, or also by working in our pre-existing
+   `interactive tutorial environment <https://try.manim.community/>`__.
+#. In the same directory containing your notebooks, you need to add a
+   file named ``Dockerfile`` with the following content:
+
+   .. code-block:: dockerfile
+
+      FROM manimcommunity/manim:v0.9.0
+
+      COPY --chown=manimuser:manimuser . /manim
+   
+   Don't forget to change the version tag ``v0.9.0`` to the version you
+   were working with locally when creating your notebooks.
+#. Make the directory with your worksheets and the ``Dockerfile``
+   available to the public (and in particular: to Binder!). There are
+   `several different options to do so 
+   <https://mybinder.readthedocs.io/en/latest/introduction.html#how-can-i-prepare-a-repository-for-binder>`__,
+   within the community we usually work with GitHub
+   repositories or gists.
+#. Once your material is publicly available, visit
+   https://mybinder.org and follow the instructions there to
+   generate an interactive environment for your worksheets.
+
+.. hint::
+
+   The repository containing our `interactive tutorial 
+   <https://try.manim.community>`__ can be found at
+   https://github.com/ManimCommunity/jupyter_examples.
+
+
+Google Colaboratory
+-------------------
+
+It is also possible to install Manim in a
+`Google Colaboratory <https://colab.research.google.com/>`__ environment.
+In contrast to Binder, where we can customize and prepare the environment
```

Comment:
Do we customize the environment for users? Don't they do that themselves? Or are you referring to try.manim.community or the Docker container?

Reply:
Not sure what I was thinking. I'll change the wording.